packages/yuujinchou/yuujinchou.2.0.0/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+synopsis: "Name pattern combinators"
+description: """
+This package implements a pattern language for selecting names. It intends to facilitate the implementation of the "import" statement or any feature that allows users to select a group of names by patterns.
+"""
+maintainer: "favonia <favonia@gmail.com>"
+authors: "favonia <favonia@gmail.com>"
+license: "Apache-2.0"
+homepage: "https://github.com/RedPRL/yuujinchou"
+bug-reports: "https://github.com/RedPRL/yuujinchou/issues"
+dev-repo: "git+https://github.com/RedPRL/yuujinchou.git"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "4.12.0" & < "5"}
```

Comment:
Is there a specific reason why this isn’t compatible with OCaml 5.0?

Reply:
I tested locally and this was compiling just fine with OCaml 5.0

Reply:
No, but I also don't know whether it would be compatible. I suppose you want me to remove the upper bound?